clients/drcachesim/tests/offline-view.templatex

```diff
@@ -5,11 +5,20 @@ Output format:
            1           0: +[0-9]+ <marker: version [0-9]>
            2           0: +[0-9]+ <marker: filetype 0x[0-9a-f]*>
            3           0: +[0-9]+ <marker: cache line size [0-9]*>
+#ifdef __ARM_FEATURE_SVE
+           4           0: +[0-9]+ <marker: vector length [0-9]* bytes>
```

Comment:
Trying to understand if it is better to add this in the header or just as one of the trace entries. Because the markers we add on prctl calls will of course not be in the header. @derekbruening

Reply:
You mean, the raw2trace code below should not look for this in a header but in its general record processing?  Then when we add prctl support we don't have to change raw2trace?

Reply:
Yes.

Reply:
I changed it to handle the marker in the general record processing.